linodecli/__init__.py

```diff
@@ -100,10 +100,16 @@ def main():
         cli.config.set_user(parsed.as_user)
 
     if parsed.version:
-        # print version info and exit
-        print("linode-cli {}".format(VERSION))
-        print("Built off spec version {}".format(cli.spec_version))
-        exit(0)
+        if not parsed.command:
```

Comment:
This is ugly, but it's basically how I would do it...

Alternatively, you could boot `--version` out of the list of known arguments, then look for it in `args`. Is that better? Is that worse? Who knows?